support/donate.md

```diff
@@ -16,7 +16,9 @@ for charitable donations.
 
 If you'd like to support our work in this way, follow the link below! Thanks!
 
-<a href="https://www.paypal.com/gb/fundraiser/charity/113209" class="large button expand round">Donate</a>
+<p style="text-align: center;">
+  <a href="https://www.paypal.com/gb/fundraiser/charity/113209" class="large button expand round" style="max-width: 30rem;">Donate</a>
```

Comment:
NAB: We're using this style inline in a bunch of places, do we want to make a utiltiy class?
e.g. For [tailwind](https://tailwindcss.com/docs/max-width#sizing-large-elements) this would be `max-w-lg` (I think)

Reply:
Nice. I went with a `md` (28rem). Though the foundation breakpoints are probably different to the tailwind ones 😬